build.gradle.kts

```diff
@@ -5,6 +5,7 @@
 plugins {
   `binary-compatibility-validator`
   `aps-plugin`
+  id("dev.zacsweers.kgp-150-leak-patcher") version "1.0.1"
```

Comment:
Is the underlying leak being tracked somewhere?

Reply:
Yep, as [KT-46368](https://youtrack.jetbrains.com/issue/KT-46368)